### DISCUSSION TITLE
How can I blur Ellipse Control by Avalonia?

### DISCUSSION THREAD

**Discussion by kyuranger** [AvaloniaUI/Avalonia#10641] [Q&A] [answered]
Previously I did this by WPF like this:
`<Ellipse Width="100" Height="100" Fill="Blue"> <Ellipse.Effect> <BlurEffect Radius="5"/> </Ellipse.Effect> </Ellipse>`

Whereas, Avalonia has not Effect property. How can I blur the control?

**Comment by timunie** [comment, marked as answer]
I think this is not yet supported out of the box. However I think you can do it using Skia. Take a look at this for example: 

- https://github.com/AvaloniaUI/Avalonia/blob/master/samples/RenderDemo/Pages/CustomSkiaPage.cs
- https://github.com/PieroCastillo/Aura.UI/blob/master/src/Aura.UI/Controls/BlurryImage/BlurryImage.cs

**Reply by kyuranger** [reply]
Thank you for answering my question.
